modules/core/misc/objc/test/objc/MatTestObjc.m

```diff
@@ -5,7 +5,7 @@
 //
 
 #import <XCTest/XCTest.h>
-#import <OpenCV/OpenCV.h>
+#import <opencv2/opencv2.h>
```

Comment:
There is an idea to rename "opencv2" => "OpenCV" framework.
Currently it is [postponed](https://github.com/opencv/opencv/pull/17165#issuecomment-635910748) for OpenCV 4.x. Should be changed with next major release OpenCV 5.0+ .
Need to check if we can change this for Swift in OpenCV 4.5.x (do we break existed user Swift code?)

BTW, this name change is handled by this [code](https://github.com/opencv/opencv/blame/e726ff3296b9e382371a778ad807d37a2e49b979/modules/objc/generator/gen_objc.py#L1434-L1435) in "build" directory.

Reply:
Yeah I was aware but when integrated via SPM, I can't use `#import <OpenCV/OpenCV.h>` because it doesn't exist in Obj-C land. I saw there's a script to do a find/replace for this, but it doesn't execute before `swift test`. I'll see if there's another workaround.

As an aside - I do like the idea to rename to `OpenCV` in 5.x. But that raises an interesting point about what will happen in the `Package.swift` after the transition, as underlying `binaryTarget` name (currently `opencv2`) might conflict with the parent/wrapper package name `OpenCV`.